server/src/main/java/org/opentosca/toscana/plugins/cloudformation/visitor/TransformModelNodeVisitor.java

```diff
@@ -171,6 +172,18 @@ public void visit(MysqlDatabase node) {
                 .allocatedStorage(allocatedStorage)
                 .storageType(storageType)
                 .vPCSecurityGroups(cfnModule.fnGetAtt(securityGroupName, "GroupId"));
+            //handle sql artifact
```

Comment:
If the sql artifact is handled here, we should remove the `TODO` under `visit(MysqlDbms node)`.